<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MAGIQ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MRS Analysis, Generation, &lt;br/&gt;and &lt;span style=&quot; font-style:italic;&quot;&gt;In-Vivo&lt;/span&gt; Quantification&lt;br/&gt;(MAGIQ)&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Software Suite&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="authorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Dickson Wong, Robarts Research Institute&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pintsButton">
     <property name="text">
      <string>PINTS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="appsButton">
     <property name="text">
      <string>APPS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fitmanButton">
     <property name="text">
      <string>FITMAN</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="spicesButton">
     <property name="text">
      <string>SPICeS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="barstoolButton">
     <property name="text">
      <string>BARSTOOL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="barstoolrvButton">
     <property name="text">
      <string>BARSTOOL (Rodent Version)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
